posit_team_contact_info.md

```diff
@@ -28,12 +28,28 @@ Authorisation for the request is required from the person's line manager, and wi
+- [ ] **Search Documentation**: Extensive online documentation with troubleshooting guides, tutorials, and user forums where you can ask questions and get answers from other users are available for Posit Workbench and specific coding problems. Some of this is internal, available on the [PHS Data Science Knowledge Base](https://public-health-scotland.github.io/knowledge-base/).
+- [ ] **Check Connectivity**: If you're experiencing connectivity issues, check your internet connection or network settings. You might need to check firewalls, VPN settings, or other network configurations. This might be the case if you're having issues connecting to other internal systems, in which case, contact DaS directly through [Service Now](https://nhsnss.service-now.com/phs/).
+- [ ] **Check Access Permissions**: Ensure that you have access permissions to the server or specific data sets. Check with someone else in your team, or DaS (via [Service Now](https://nhsnss.service-now.com/phs/)) to confirm your access permissions. _All databases connections that were available on the old server infrastructure have been migrated and made available for testing during UAT._
+- [ ] **Check Browser**: Sometimes, the issue might be related to the browser you are using. This has been identified in a couple of areas already with Mozilla Firefox. Try using a different browser to see if the issue persists, all Chromium based browsers are tested and work as expected (i.e. Microsoft Edge and Google Chrome).
 
-The team will respond to all queries and requests for support submitted via this form, and will always respond as quickly as possible.
+If you have followed these troubleshooting steps and the issue still persists, please raise a support request by providing the following details (you can also use this form to reach out with questions about the infrastructure) :
 
-Queries and requests for support with Posit applications cannot be made directly in Service Now by users, unless explicitly authorised by the Data Science team.
+- Specific steps taken that led to the issue
+- Error messages or symptoms
+- Steps you have already taken to resolve the issue
+- Any recent changes made to your system or the server
+- Any relevant log files or screenshots that can help identify the issue
 
-**Please do not raise a request or incident in Service Now unless requested to by a member of the Data Science team.**
+Please complete the form at [https://forms.office.com/e/pqmmNMkhT6](https://forms.office.com/e/pqmmNMkhT6).
+
+The team aim to respond to all queries and requests for support submitted via this form as quickly as possible.
+
+**Queries** and **requests** for support with Posit applications cannot be made directly in Service Now by users, unless explicitly authorised by the Data Science team.
```

Comment:
I just wondered here, whether for monitoring purposes, we really need to emphasise that queries and requests additionally cannot be emailed as we have no way of monitoring, that the forms are there to enable the requests to go thru the correct channel, or some better phrase than that?

Reply:
I agree with Clive on this one.